packages/core/src/Modules/Cashier/Components/payment-agent-list.jsx

```diff
@@ -154,6 +162,7 @@ PaymentAgentList.propTypes = {
 };
 
 export default connect(({ modules }) => ({
+    error: modules.cashier.config.payment_agent.error,
```

Comment:
isn't the error coming on verify_email? why are we checking the error in payment agent container?

Reply:
I receive the error message while I am at `payment_agent` container. That's why I am retrieving it from payment agent.

Reply:
but this means any other error that we set for payment agent will hide the entire page and show that, and we don't want that. we only want to hide the entire page on verify email errors.